contracts/risc0-recursion/src/bin/guest.rs

```diff
@@ -0,0 +1,16 @@
+#![no_main]
+#![no_std]
+
+extern crate alloc;
+
+use sdk::guest::env;
+
+risc0_zkvm::guest::entry!(main);
+
+fn main() {
+    let input: alloc::vec::Vec<risc0_recursion::ProofInput> = env::read();
+    input.iter().for_each(|input| {
+        risc0_zkvm::guest::env::verify(input.image_id, &input.journal)
+            .expect("Verification failed");
```

Comment:
I feel like what we're doing here is verifying multiples proofs and output only one proof out of them. (How can we retrieve output of each proofs ?)
That implies that for each blob, we'll _have_ to generate a proof. What I had in mind when we talked about recursion was to _execute_ multiple blob at once, and generate only one proof out of all blobTxs. 
--> We'll have 1 proof generated for x BlobTxs; here I feel like we'll have to generate x proofs, and +1 for the recursed one